doc/reference/autoinstall-reference.rst

```diff
@@ -1190,6 +1190,50 @@ Example:
       kernel:
         flavour: hwe
 
+.. _ai-kernel-crash-dumps:
+
+kernel-crash-dumps
+~~~~~~~~~~~~~~~~~~
+
+* **type:** mapping, see below
+* **default:** see below
+* **can be interactive:** no
+
+Toggle kernel crash dumps enablement.
+
+The default configuration will result in dynamic enablement of kernel crash dumps as described in the following document (doesn't exist yet).
+
+Default configuration:
+
+.. code-block:: yaml
+
+   autoinstall:
+      # Allow kernel crash dumps to be enabled dynamically.
```

Comment:
only 24.10 sees this default behavior, pre-24.10 is default off, let's spell that out.

Reply:
Good point. Done!